src/ansible_compat/runtime.py

```diff
@@ -77,6 +83,29 @@ def __init__(
             raise RuntimeError(
                 f"Found incompatible version of ansible runtime {self.version}, instead of {min_required_version} or newer."
             )
+        if require_module:
+            self._check_module()
+
+    def _check_module(self) -> None:
+        """Assure that Ansible Python module is installed and matching CLI version."""
+        ansible_module_version = None
+        try:
+            # pylint: disable=import-outside-toplevel
+            from ansible.release import __version__
```

Comment:
For dynamic imports, I'd prefer using importlib. It also makes it easier to inject `ImportError`